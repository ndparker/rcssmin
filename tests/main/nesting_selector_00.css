/*! Issue 17 */
.box {
    & :is(ol) {
        margin: 0;
    }

    &.foo {
        duh: 2px;
    }

    &/**/.bar {
        duh: 2px;
    }

    &/*! hey! */.plop {
        duh: 2px;
    }
    &/**/ /*!duh*/.baz {
        duh: 2px;
    }
    &/**/ /*duh*/.zopp {
        duh: 2px;
    }
}

&
