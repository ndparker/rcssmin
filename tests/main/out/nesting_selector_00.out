.box{& :is(ol){margin:0}&.foo{duh:2px}&.bar{duh:2px}&.plop{duh:2px}& .baz{duh:2px}& .zopp{duh:2px}}&